convert2rhel/pkghandler.py

```diff
@@ -1057,3 +1057,21 @@ def _parse_pkg_with_dnf(pkg):
 
     pkg_ver_components = (name, epoch, version, release, arch)
     return pkg_ver_components
+
+
+def get_highest_package_version(pkgs):
+    """
+    Get the highest version from the provided list of packages.
+    :param pkgs: A tuple containing the name of the package list (as a string) and the list of package versions (as a list of strings).
+    :type pkgs: tuple(str,list[str])
+    :return: Return a single package with the highest version.
+    :rtype: str
+
+    :raises ValueError: If the list is empty, raise a ValueError.
+    """
+    try:
+        return max(pkgs[1])
+    except ValueError:
+        loggerinst.debug("The list of %s packages is empty." % pkgs[0])
+
+    return
```

Comment:
I would vote to continue without the return statement. It does not add anything to the current implementation. 

Also, not needed to explicitly say that it returns `None`. An empty return will always be `None`

Reply:
Going by this, and it makes sense IMO. When returns have some explicit return then we should be explicit everywhere

https://github.com/oamg/convert2rhel/security/code-scanning/926

Reply:
I think I'll go with the exception.
Given the context where it's used, if the list is indeed empty, there is something very wrong, and it should raise the exception.
If we ever decide to use the function elsewhere, it can be refactored